md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/BatchAPI.java

```diff
@@ -87,20 +86,17 @@ RestWrapper get(
      * @return nothing.
      */
     @RequestMapping(value = "/{id}", method = RequestMethod.DELETE)
-    public
-    @ResponseBody
+    @ResponseBody public
     RestWrapper delete(
-            @PathVariable("id") Long batchId, Principal principal,
-            ModelMap model) {
+            @PathVariable("id") Long batchId, Principal principal) {
 
         RestWrapper restWrapper = null;
         try {
             batchDAO.delete(batchId);
-            // s.delete("call_procedures.DeleteBatch", batch);
-
             restWrapper = new RestWrapper(null, RestWrapper.OK);
-            LOGGER.info("Record with ID:" + batchId + " deleted from Batch by User:" + principal.getName());
+            LOGGER.info(RECORDWITHID + batchId + " deleted from Batch by User:" + principal.getName());
         } catch (Exception e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1166)